pay-api/tests/unit/services/test_hashing_service.py

```diff
@@ -0,0 +1,32 @@
+# Copyright © 2019 Province of British Columbia
```

Comment:
Please add a test_direct_pay_service as it involves more logic.

Reply:
working on it.